### PR TITLE
Meet/refactor id file names

### PR DESCRIPTION
**Pull request by pandyama**
For layout IDs, I used camelCase convention.
For xml File names I used a convention I read about online.
https://medium.com/mindorks/android-resource-naming-convention-42e4e8026614
https://chaseadams.io/posts/most-common-programming-case-types/

Compiled it and ran to make sure all the functionalities are working just as before.

### CODE REVIEW

**Comment by oyiza** [comment]
so from my experience with naming files and variables, it's usually safe to go with lowercase whenever possible (unless it's convention for the framework) eg: `PascalCase` naming for kotlin files and `snake_case` naming for layout files. There's also `camelCase` naming for ids and variables. From the links you posted, those seem to agree with that too.

also, for the id naming just pointing out that to me, `btnDeleteJob` reads better than `deleteJobBTN` and the second makes it look like the emphasis is on "BTN" and not "deleteJob". just my thoughts at the moment but even something like `deleteJobBtn` is better (it's camelCase)

**Comment by pandyama** [comment]
Yaa sure thing, i'll change those IDs to follow `deleteJobBtn` pattern

**Comment by pandyama** [comment]
@oyiza updated IDs now including the mobile_navigation.xml file, Let me know if its good

File: app/src/main/res/navigation/mobile_navigation.xml [hunk 1]
```diff
@@ -3,28 +3,28 @@
...
-        android:id="@+id/navigation_addjob"
+        android:id="@+id/navigationAddJobFragment"
         android:name="com.example.oddhours.ui.addjob.AddJobFragment"
         android:label="@string/title_add_job"
-        tools:layout="@layout/fragment_addjob" />
+        tools:layout="@layout/fragment_add_job" />
 
     <fragment
-        android:id="@+id/shifts"
+        android:id="@+id/shiftsFragment"
```
Comment (oyiza) [inline app/src/main/res/navigation/mobile_navigation.xml]: maybe `navigationShiftsFragment`?

Comment (pandyama) [reply app/src/main/res/navigation/mobile_navigation.xml]: oh yes lol, mb... im goofing up right now

**Comment by oyiza** [comment]
looks good man. it was a lot of changes haha

**Comment by pandyama** [comment]
> looks good man. it was a lot of changes haha

yaa so many because of the references in Kotlin files

**Pull request merged**